ctru-rs/Cargo.toml

```
@@ -15,6 +15,6 @@ ctru-sys = { path = "../ctru-sys", version = "0.4" }
 const-zero = "0.1.0"
 linker-fix-3ds = { git = "https://github.com/Meziu/rust-linker-fix-3ds.git" }
 pthread-3ds = { git = "https://github.com/Meziu/pthread-3ds.git" }
-libc = { git = "https://github.com/Meziu/libc.git" }
+libc = { version = "0.2" }
```

If this is just going to specify the version, this should be:
```suggestion
libc = "0.2"
```